Makefile

```diff
@@ -105,16 +105,16 @@ test: manifests generate fmt vet test-unit test-e2e #HELP Run all tests.
 .PHONY: e2e
 FOCUS := $(if $(TEST),-v -focus "$(TEST)")
 E2E_FLAGS ?= ""
-e2e: $(GINKGO) #EXHELP Run the e2e tests.
-	$(GINKGO) --tags $(GO_BUILD_TAGS) $(E2E_FLAGS) -trace -progress $(FOCUS) test/e2e
+e2e: $(SETUP_ENVTEST) #EXHELP Run the e2e tests.
```

Comment:
Thank you for making this self-contained, nobody has to remember how to use setup-envtest anymore!